.github/workflows/icu4c.yml

```diff
@@ -376,10 +376,10 @@ jobs:
             build_flags: '/p:Configuration=Debug /p:Platform=Win32'
           - test_flags: 'arm Release'
             build_flags: '/p:Configuration=Release /p:Platform=ARM'
+          - test_flags: 'x64 Release cpplatest'
+            build_flags: '/p:OverrideLanguageStandard=stdcpplatest /p:Configuration=Release /p:Platform=x64'
           - test_flags: 'x64 Release'
-            build_flags: '/p:LanguageStandard=stdcpplatest /p:Configuration=Release /p:Platform=x64'
-          - test_flags: 'x64 Release'
-            build_flags: '/p:_HAS_EXCEPTIONS=0 /p:Configuration=Release /p:Platform=x64'
```

Comment:
@eggrobin I am going to merge, but I did want to ask whether removing `/p:_HAS_EXCEPTIONS=0` was intentional. Was that another unrecognized option?

Reply:
Yes. This is a substring of PreprocessorDefinitions in ClCompile (cf. LanguageStandard which is also in ClCompile), not a property. And it is set unconditionally : https://github.com/unicode-org/icu/blob/aef7ad283d5e64ba2c7172f03873da526ddacad5/icu4c/source/allinone/Build.Windows.Library.ProjectConfiguration.props#L11.